controllers/host/host_controller.go

```diff
@@ -667,7 +667,8 @@ func (r *HostReconciler) ReconcileInitialState(client *gophercloud.ServiceClient
 	desiredState := profile.AdministrativeState
 
 	if desiredState != nil && *desiredState != host.AdministrativeState &&
-		instance.Status.DeploymentScope == cloudManager.ScopeBootstrap {
+		instance.Status.DeploymentScope == cloudManager.ScopeBootstrap &&
+		!r.CloudManager.GetStrategySent() {
```

Comment:
Why do we want to check the strategy here? Isn't the check of the scope sufficient?

Reply:
No, in case of network reconfiguration, scope is not changed for host, it will only be updated for platformnetwork so it'll not help.

Reply:
acked